spec/services/json_species_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+RSpec.describe "JsonSpecies", type: :service do
+
+  before(:all) do
+    json_file = File.open(Rails.root.join("lib/data/Species.json"))
+    @json_species_array = JsonSpecies.to_a(json_file)
+    @json_species = @json_species_array.first
+  end
+
+  it "should create 2 JsonSpecies objects from the json" do
+    expect(@json_species_array.count).to eql 69
```

Comment:
This is weird, in the spec description it says "it should create 2 JsonSpecies"  but you're checking against 69, which is it?